extensions/actions/base_test.py

```diff
@@ -19,9 +19,9 @@
 from core.domain import action_registry
 from core.tests import test_utils
 
-EXPLORATION_START_ID = 'ExplorationStart'
-ANSWER_SUBMIT_ID = 'AnswerSubmit'
-EXPLORATION_QUIT_ID = 'ExplorationQuit'
+EXPLORATION_START_TYPE = 'ExplorationStart'
```

Comment:
ACTION_TYPE_EXPLORATION_START
ACTION_TYPE_ANSWER_SUBMIT
ACTION_TYPE_EXPLORATION_QUIT

Reply:
Btw for this and the file below, why are these constants being defined in tests? Shouldn't we be using the canonical references for these constants (like feconf or something)?

Unless your explicit purpose is to mock these, but in that case, it should be something like ACTION_TYPE_FAKE_ACTION_FOR_TESTS to make the intention clear.

Reply:
Done.